site/ar/tutorials/text/nmt_with_attention.ipynb

```diff
@@ -0,0 +1,1136 @@
+        "  </td>\n",
+        "  <td>\n",
+        "    <a href=\"https://storage.googleapis.com/tensorflow_docs/docs/site/en/tutorials/text/nmt_with_attention.ipynb\"><img src=\"https://www.tensorflow.org/images/download_logo_32px.png\" />نزّل الدّفتر</a>\n",
+        "  </td>\n",
+        "</table>"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "ExecuteTime": {
+          "end_time": "2020-03-26T09:16:19.603017Z",
+          "start_time": "2020-03-26T09:16:19.595217Z"
+        }
+      },
+      "source": [
+        "<div dir=\"rtl\">\n",
+        "\n",
+        "Note: قامت مجموعة المتطوعين في مشروع Tensorflow بترجمة هذا المحتوى. نظرًا لأن هذه الترجمات تعتمد على قاعدة أقصى الجهد (best-effort) ، فلا نضمن أنها انعكاس دقيق وحديث [للمحتوى الرسمي باللغة الإنجليزية](https://www.tensorflow.org/?hl=en). إذا كانت لديك اقتراحات لتحسين هذه الترجمة ، يرجى إرسال \"Pull request\" إلى مشروع [tensorflow/docs-l10n](https://github.com/tensorflow/docs-l10n). للتطوع أو مراجعة ترجمات المجموعة يرجى ارسال ايميل إلى docs@tensorflow.org.\n",
+        "\n",
+        "</div>"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "CiwtNgENbx2g"
+      },
+      "source": [
+        "<div dir=\"rtl\">\n",
+        "\n",
+        " نقوم في هذا الدفتر بتعلّم كيفية تدريب نموذج تسلسل إلى تسلسل (seq2seq) للترجمة من الإسبانية إلى الإنجليزية. هذا مثال متقدم يفترض بعض المعرفة بنماذج التسلسل إلى تسلسل. بعد تدريب النموذج في هذا الدفتر، ستتمكن من إدخال جملة إسبانية ، مثل: \n",
+        "\n",
+        "</div> \n",
+        "\n",
+        "\"*¿todavia estan en casa؟*\"  \n",
+        "\n",
+        "\n",
+        "<div dir=\"rtl\">  واستخراج الترجمة الإنجليزية:\n",
+        "</div>  \n",
+        "\n",
+        "\"*are you still at home?*\"\n",
+        "\n",
+        "<div dir=\"rtl\">\n",
+        "\n",
+        "جودة الترجمة مقبولة لمثال بسيط ، ولكن ربما يكون الرسم البياني للانتباه (attention) الناتج أكثر إثارة للاهتمام. فهو يوضح  أجزاء الجملة المدخلة التي لفتت انتباه النموذج أثناء الترجمة:\n",
+        "\n",
+        "</div>\n",
+        "\n",
+        "<img src=\"https://tensorflow.org/images/spanish-english.png\" alt=\"spanish-english attention plot\">\n",
+        "\n",
+        "<div dir=\"rtl\">\n",
+        "\n",
+        "Note: \n",
+        "يستغرق هذا المثال حوالي 10 دقائق للتشغيل على GPU P100.\n",
+        "\n",
+        "</div> "
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 0,
+      "metadata": {
+        "colab": {},
+        "colab_type": "code",
+        "id": "tnxXKDjq3jEL"
+      },
+      "outputs": [],
+      "source": [
+        "from __future__ import absolute_import, division, print_function, unicode_literals\n",
+        "\n",
+        "try:\n",
+        "  # %tensorflow_version only exists in Colab.\n",
+        "  %tensorflow_version 2.x\n",
+        "except Exception:\n",
+        "  pass\n",
```

Comment:
```suggestion
```

Now that Colab defaults to 2.2 and we're Py3, we'll remove __future__ imports and %tensorflow_version in notebooks